test/integrations/destinations/http/common.ts

```diff
@@ -158,27 +158,27 @@ const destinations: Destination[] = [
       propertiesMapping: [
         {
           from: '$.event',
-          to: '$.event',
+          to: '$.body.event',
```

Comment:
didn't understood the scenario here - why the `body` prefix ?

Reply:
XML cannot have more the one root tag. So update the mapping to have one root key `body`.  Added a negative test for this as well.